cmd/create/cluster/cmd.go

```diff
@@ -2067,6 +2266,382 @@ func run(cmd *cobra.Command, _ []string) {
 			r.Reporter.Errorf("%s", err)
 			os.Exit(1)
 		}
+
+		// Autoscaler configurations
+		isClusterAutoscalerSet, err = interactive.GetBool(interactive.Input{
```

Comment:
Any way to consider this as set without going through interactive?

Reply:
Should be resolved same as above, let me know if this does not work.

Reply:
I'm changing the implementation a bit, because currently it doesn't let the user to set the variables interactively unless one of them were passed at the CLI